projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -64,12 +64,19 @@ export class NavigationUIComponent implements OnInit, OnDestroy {
   private openNodes: HTMLElement[] = [];
   private subscriptions = new Subscription();
   private resize = new EventEmitter();
+  private arrowControls: Subject<KeyboardEvent> = new Subject();
```

Comment:
Lets use the `protected` modifier here and in general across the spartacus project. The reason is that `protected` shows that it is not intended for direct manipulation by lib users. But they can if they really need to. 
If we make these `private`, then it is not possible for them to do so without making a copy of the spartacus library so that they can modify it (its really annoying to do that).

Reply:
done ;)